.github/workflows/ghcr_runtime.yml

```diff
@@ -1,5 +1,5 @@
 # Workflow that builds, tests and then pushes the runtime docker images to the ghcr.io repository
-name: Build, Test and Publish Runtime Image
+name: Build, Test and Publish RT Image
```

Comment:
any particular reason for switching to `RT`? I think new contributors might find `Runtime` easier to grok

Reply:
Mainly the long job names push out the running time of the job in the below workflow list, e.g.
`Build, Test and Publish RT Image / RT Unit Tests (Root) (nikolaik) (pull_request)     Successful in 11m`
The latter part gets easily pushed out.
Also, I think RT is a quite common abbrevation for runtime in IT.